app/src/main/java/com/example/encoratask/model/Character.kt

```diff
@@ -0,0 +1,46 @@
+package com.example.encoratask.model
+
+import org.json.JSONObject
+import java.io.Serializable
+
+/**
+ * Created by Erick Sanchez
+ * Revision 1 - 09/06/21
+ */
+class Character:Serializable {
```

Comment:
I would recommend to have data class instead of a POJO (in Kotlin, I mean... POKO?). In the way that it is right now, you must implement at least .equals() method or you're wasting memory adding and removing UI elements in the adapter

Reply:
I dont understand this, jeje

Reply:
basically you're using the Java way to build your Model. Kotlin has this data classes to avoid you to implement certain methods to compare if an object is already in the list. For instance, using the right implementation of List and a data class you can freely say list.remove(element) without iterate, remove, create temp lists, overwrite the original list, etc

Reply:
Ok bro, I got it, thanks bro

Reply:
Also in Android, use Parcelable instead of Serializable since its optimized

Reply:
Sounds good to me